source/release-notes/swift.txt

```diff
@@ -0,0 +1,14 @@
+===================
+Swift SDK Changelog
+===================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: twocols
+
+You can find the `Swift SDK Release Notes <https://github.com/realm/realm-swift/releases>`__
```

Comment:
Hmm. The link to the Swift SDK Release Notes exists in the Swift SDK sidebar and it's got the extlink icon showing in the ToC. Now that you're adding this page, I wonder if that release notes entry in the ToC should link to here, instead, and then it won't be an extlink? (And presumably the same for the other SDKs?)

Reply:
We mostly added these pages because users got confused when they looked in "release notes" and didn't see any... SDK release notes. I think it would be sort of annoying to click the link in the Swift SDK sidebar and end up here... and snooty doesn't let you put a page in multiple parts of the sidebar anyway ;)